lib/qa/authorities/linked_data/config.rb

```diff
@@ -39,6 +39,10 @@ def prefixes
         @prefixes ||= authority_config.fetch(:prefixes, {})
       end
 
+      def service_uri
+        @service_uri ||= authority_config.fetch(:service_uri, nil)
```

Comment:
service_uri is used to locate the triple holding total number of search results.